header.php

```diff
@@ -41,6 +41,7 @@ function searchFocus() {
 	<body>
 		<div class="topBar">
 			<a href="/" style="margin-right: 8px;color:white;">OMDB - osu! map database</a>
+			<a><?php echo getenv("PUBLIC_URL"); ?></a>
```

Comment:
Is this something left over from testing?

Reply:
Yea oops